src/compiler/build/write-build.ts

```diff
@@ -21,11 +31,10 @@ export const writeBuild = async (config: d.Config, compilerCtx: d.CompilerCtx, b
 
     // successful write
     // kick off writing the cached file stuff
-    // await compilerCtx.cache.commit();
```

Comment:
just deleting commented out code here

Reply:
There's a PR open which uncomments these lines (and makes a few other changes): https://github.com/ionic-team/stencil/pull/3456

We may want to take a look at that at some point, I think it's fine to remove these lines for now though